.ado/jobs/cli-init.yml

```diff
@@ -170,8 +170,7 @@ jobs:
             additionalInitArguments:
             additionalRunArguments:
             continueOnBuildFailure: true # See issue #6129
-      pool:
-        vmImage: $(VmImage)
+      pool: $(AgentPool_medium)
```

Comment:
So the medium pool is just for the cli init CI?

Reply:
Yes, experimenting here with how it affects overall time, at least it would be 16 gig of ram so the WinUI3 WinRT stuff won't blow through the agents ram at the best :)

Reply:
To get multicore build to work I had to update the VM's to 4c32gb because with 4c16gb did not have enough RAM and was actually slower than the hosted 2c7gb vm's.